source/assets/stylesheets/styles/_page_header.scss

```diff
@@ -49,3 +49,11 @@
     font-size: 2.4em;
   }
 }
+
+.page-header__subtitle {
+  @include responsive-sub-title;
```

Comment:
Actually I couldn't use `responsive-text` as the desktop font size was too big for contact page. I improved all texts to be responsive though here: bf218bf